### PR TITLE
[Fix-16740][SeaTunnel-Task] fix can't submit resource center config file issue

### LINKED ISSUE #16740: [Bug] [Task][SeaTunnel] Some Issue about seatunnel task
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

I try to use seatunnel to submit job via DS.
I find there has some case can be improve.

I defined a seatunnel task, the config file is from resource center.
image

When I run this task, I find it has some issue.
First, the config file location will be delete the first charsert
image
It will update the shell script to 
```
${SEATUNNEL_HOME}/bin/seatunnel.sh --config dfs://nameservice1/dolphinscheduler/root/resources/lnj/batch.conf --deploy-mode local
```
This will obviously be an error, because `hdfs` update to `dfs`.
In local env, the file path will replace from `file:/xxx` to `ile:/xxx`

In my understanding seatunnel is not support read file from HDFS or other file system now. 
So even we fix this issue, it still can not run well.

I think we can download the file from resource center to local, (like the HiveCli Task). then the submit script use the local file path.

Another thing i noticed is when i choose file, the demo config still will be write to taskParams. We can improve this as it's useless.

image

### What you expected to happen

bug fix

### How to reproduce

create an seatunnel task, config file using resource center

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liunaijie**
## Purpose of the pull request
close  #16740

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by liunaijie** [comment]
image

image


All failures look like timeout issues, can someone help to re-trigger the CI? thanks

**Comment by SbloodyS** [comment]
> All failures look like timeout issues, can someone help to re-trigger the CI? thanks

Done.

File: dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java [hunk 1]
```diff
@@ -154,26 +155,21 @@ private String buildCommand() throws Exception {
 
     protected List<String> buildOptions() throws Exception {
         List<String> args = new ArrayList<>();
+        args.add(CONFIG_OPTIONS);
```
Comment (SbloodyS) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java:158]: Please add some UT for this.

Comment (liunaijie) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java:158]: Done, Update the SeatunnelTaskTest, PTAL, thanks

**Comment by liunaijie** [comment]
re-trigger CI

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by davidzollo — APPROVED** [review]
+1
good job

**Pull request merged**